### DISCUSSION TITLE
[New] Désactiver la lecture automatique des vidéos et sons

### DISCUSSION THREAD

**Discussion by ACTLEM** [cnumr/best-practices#34] [Bonnes pratiques] [answered]
L'activation automatique des vidéos et des sons (auto play) au chargement des pages web implique une utilisation de ressources supplémentaires et pas nécessairement utiles sur chaque tiers (terminal, réseau, datacenter).
D'autre part, elle attire l'attention de l'utilisateur et peut le distraire du service recherché à l'origine.
Ces lectures doivent se faire à la demande de l'utilisateur.

**Comment by arnolem** [comment]
A fusionner avec ça : https://github.com/cnumr/best-practices/discussions/46

**Comment by davidgroult** [comment]
Point essentiel! Tout à fait d'accord pour la fusion/ajout.

**Comment by ACTLEM** [comment]
Effectivement la fusion semble pertinente car on peut effectivement désactiver un autoplay sans pour autant désactiver le préchargement.

**Comment by ACTLEM** [comment]
Note équipe projet: Bien ajouter dans cette BP qu'il faut aussi désactiver le préchargement

**Comment by ACTLEM** [comment, marked as answer]
https://github.com/cnumr/best-practices/issues/135
